app/views/ViewAllTemplatesView.scala.html

```diff
@@ -64,6 +63,20 @@
         }
         }
 
+        @request.flash.get(TFE_DELETED_DRAFT_TEMPLATE_ID).map { templateName =>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    @notificationBanner(
+                        NotificationBanner(
+                            title = messages("viewAllTemplates.deletedNotification.title"),
+                            content = messages("viewAllTemplates.deletedNotification.content", templateName),
+                            classes = "govuk-notification-banner--success"
```

Comment:
The success banner width is not displayed correctly when the last template has been deleted.  see screenshot
<img width="814" alt="Screenshot 2024-09-17 at 08 21 42" src="https://github.com/user-attachments/assets/e8d5f4ee-a0c0-42d8-8f61-3642ac6d0203">

Reply:
The `ViewAllTemplatesView` only sets full width content if there are templates else it sets it to two thirds